src/py42/exceptions.py

```diff
@@ -167,6 +167,13 @@ def __init__(self, exception, message=None):
         super(Py42UserAlreadyExistsError, self).__init__(exception, message)
 
 
+class Py42UserDoesNotExist(Py42NotFoundError):
+    """An exception raised when user does not exist in the detection list."""
```

Comment:
I'd also modify the doc to say `... when user is not in the detection list` rather than `not exist`

Reply:
changed.